Automattic/jetpack
# My Jetpack: Use Jetpack Modules

Fixes a fatal error introduced in #33283

Using Jetpack class directly works when Jetpack plugin is active, but without the plugin this throws fatals - for example in standalone plugins like Jetpack Boost.

### Other information:

## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
1. Use trunk
2. Deactivate Jetpack
3. Visit my-jetpack, verify fatal error
4. Apply patch
5. Visit my-jetpack, verify no error
